develop-guide/writeblitz/writeblitz.rst

```diff
@@ -246,6 +246,60 @@ other existing apis.
         function: get_traffic_stream_objects
         ...
 
+rest
+____
+
+The ``rest`` action allows you to make rest call to any endpoint on a device. Rest uses http protocols to 
+transfer data. Five http protocols are supported, `get`, `post`, `put`, `patch` and `delete`.
+
+You can find additional information on rest, using this `tutorial
+<https://pubhub.devnetcloud.com/media/genie-feature-browser/docs/#/apis>`_.
+
+.. code-block:: YAML
+
+    test_sections:
+        - plain_actions:
+            - rest:
+                method: get
+                dn:  '/api/mo/sys/intf/phys-[eth1/1].json'
+                device: N93_3
+            - rest:
+                method: delete
+                device: N93_3
+                dn: '/api/mo/sys/bgp/inst.json'
+            - rest:
+                method: put
+                dn:  '/api/mo/sys/bgp/inst/dom-default/af-ipv4-mvpn.json'
+                device: N93_3
+                payload: {
+                    "intf-items": {
+                      "phys-items": {
+                        "PhysIf-list": [
+                          {
+                            "adminSt": "down",
+                            "id": "eth1/2",
+                            "userCfgdFlags": "admin_layer,admin_state"
+                          }
+                        ]
+                      }
+                    }
+                  }
+
+You can also change your connection alias from rest to anyone of the following 
+['cli', 'yang', 'rest', 'xml', 'web', 'restconf', 'netconf', 'gnmi'] and still take advantage 
```

Comment:
????

This only works for Rest...why are you bringing these?

Reply:
But then why do we set connection_alias in the code ?

Reply:
with cli, `still take advantage` what does it mean?

Reply:
cause you can use any custom one they want,   but these are mapping datafile.